### PR TITLE
Correct instancenorm input size

### LINKED ISSUE #4170: `InstanceNorm1d` do not support 2d input
Ref [doc](http://pytorch.org/docs/master/nn.html#torch.nn.InstanceNorm1d), the input only supports `(N, C, L)` format. This is not consistent with [BatchNorm1d](http://pytorch.org/docs/master/nn.html#torch.nn.BatchNorm1d) which supports also `(N, C)`.

### PR DESCRIPTION
**Pull request by innerlee**
fix https://github.com/pytorch/pytorch/issues/4170.

### CODE REVIEW

**Comment by soumith** [comment]
thank you for the quick doc fix!
